### ISSUE TITLE
Simulator always runs in portrait mode

### ISSUE THREAD

**Issue by yadav-rahul** [BuildmLearn/BuildmLearn-Toolkit-Android#378]
As per now `Simulator` runs in portrait mode only and is independent of device orientation but IMO for testing it should support landscape mode too.

@croozeus @opticod Am I missing something here or It's supposed to be Simulator's default behaviour?

**Comment by yadav-rahul** [comment]
@opticod Please share your view on this?

**Comment by opticod** [comment]
Definitely, you can add the landscape mode too. @yadav-rahul You can proceed
